content/en/dashboards/template_variables.md

```diff
@@ -76,6 +76,11 @@ After the widget is saved, the value of the template variable is the one selecte
 
 {{< img src="dashboards/template_variables/selecting_template_variables.png" alt="Selecting template variables"  style="width:75%;">}}
 
+#### Associated Template Variables
+When selecting a template variable value, you will see two sections for **Associated Values** and **Other Values**. Associated Values are calculated by taking into  the other template variable values selected on the page, and seamlessly identifies the related values without any configuration. 
```

Comment:
```suggestion
When selecting a template variable value, the **Associated Values** and **Other Values** sections are displayed. Associated values are calculated by taking into  the other template variable values selected on the page, and seamlessly identifies the related values without any configuration. 
```

@MirandaKapin is there a word missing in the second sentence? Also, what does "other values" refer to? seems like only associated values are being explained here.